worker/uniter/uniter.go

```diff
@@ -444,7 +450,7 @@ func (u *Uniter) loop(unitTag names.UnitTag) (err error) {
 			StartRetryHookTimer: retryHookTimer.Start,
 			StopRetryHookTimer:  retryHookTimer.Reset,
 			Actions: actions.NewResolver(
-				u.logger.Child("actions"),
+				u.logger.Child("expectedActions"),
```

Comment:
Why change this? The original had the logger be "juju.worker.uniter.actions" which is what was there before, why change it?

Reply:
didn't intend to, maybe a search/replace fail